WTWSMS/common/traits/VIET_Immersion_Other.txt

```diff
@@ -45,7 +46,7 @@ griot = { #West African Griot
 
 	customizer = no
 	
-	cannot_inherit = yes
+	cannot_inherit = no
```

Comment:
I think my research is good, but if you want more research, I'll take out this Trait change now so we can implement what has been approved of.